launcher/util.go

```diff
@@ -3,11 +3,22 @@ package launcher
 import (
 	"context"
 	"fmt"
+	"io"
 
+	"github.com/google/go-tpm/legacy/tpm2"
 	"google.golang.org/api/impersonate"
 	"google.golang.org/api/option"
 )
 
+// TPMDAParams holds TPM Dictionary Attack parameters
+type TPMDAParams struct {
+	LockoutCounter      uint32
+	MaxAuthFail         uint32
+	LockoutInterval     uint32
+	LockoutRecovery     uint32
```

Comment:
here and everywhere else, why does this not match the spec values: `maxTries`, `recoveryTime`, `lockoutRecovery`

Reply:
There are two sets of naming, in TPM source code and tss library, they represents the same thing but somehow use the different naming.
https://github.com/google/go-tpm/blob/364d5f2f78b95ba23e321373466a4d881181b85d/legacy/tpm2/test/tpm2_test.go#L2356